packages/api-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tupaia/api-client",
-  "version": "2.1.1",
+  "version": "3.0.0",
```

Comment:
Does it trigger anything to run?

Reply:
No, we have to do a manual publish to npm. Given that nobody other than Tamanu is actually using the client though, I'm not too sure what the protocol is. I was planning on publishing a new version all the same after the code is released 👍